build/templates/_converters.py.mako

```diff
@@ -8,6 +8,32 @@ import datetime
 import six
 
 
+def _string_to_list(string, prefix):
```

Comment:
It might sense to reference public documentation on the allowable syntax. I haven't looked for something generic but found [this](http://zone.ni.com/reference/en-XX/help/370736U-01/nidcpowerviref/nidcpower_initialize_with_channels/):

> Specify multiple channels by using a channel list or a channel range. A channel list is a comma (,) separated sequence of channel names (for example, 0,2 specifies channels 0 and 2). A channel range is a lower bound channel followed by a hyphen (-) or colon (:) followed by an upper bound channel (for example, 0-2 specifies channels 0, 1, and 2). In the Running state, multiple output channel configurations are performed sequentially based on the order specified in this parameter.

So at the very least, you are missing `:` as a delimiter. Also make sure you have tests that validate that ordering is preserved based on the last part.